MusicCatalog.RazorPagesView/Pages/Songs/SongsIndex.cshtml

```diff
@@ -6,101 +6,39 @@
-    .action-buttons button:hover {
-        background-color: #0056b3;
-    }
-    
-    .action-buttons .btn-delete{
-        background-color: #dc3545;
-    }
-    
-    .duration-cell {
-        font-weight: bold;
-    }
-    
-</style>
-
-<div class="text-center page-container">
-    <h1 class="page-title">Songs</h1>
-    <a asp-page="./CreateSong" class="create-button">Create</a>
-    <form method="post">
-        <table>
-            <thead>
-                <tr>
-                    <th>Title</th>
-                    <th>Duration</th>
-                    <th>Release Date</th>
-                    <th>Genre</th>
-                    <th>Artists</th>
-                    <th colspan="2" class="text-center">Actions</th>
-                </tr>
-            </thead>
-            @foreach (var song in Model.Songs)
-            {
-                <tr>
-                    <td>@song.Title</td>
-                    <td class="duration-cell">@($"{song.Duration / 60:D2}:{song.Duration % 60:D2}")</td>
-                    <td>@song.ReleaseDate.ToShortDateString()</td>
-                    <td>@song.Genre?.Name</td>
-                    <td>@string.Join(", ", song?.Artists ?? Enumerable.Empty<Domain.Models.Artist>())</td>
-                    <td class="action-buttons">
-                        <a asp-page="./UpdateSong" asp-route-id="@song?.Id">Update</a>
-                    </td>
-                    <td class="action-buttons">
-                        <button type="submit" asp-page-handler="delete" asp-route-id="@song?.Id" class="btn-delete">Delete</button>
-                    </td>
-                </tr>
-            }
-        </table>
-    </form>
+<div class="text-center songs-page">
+    <div class="page-container">
+        <h1 class="page-title">Songs</h1>
+            <a asp-page="./CreateSong" class="create-button">Create</a>
+            <form method="post">
+                <table>
+                    <thead>
+                    <tr>
+                        <th>Title</th>
+                        <th>Duration</th>
+                        <th>Release Date</th>
+                        <th>Genre</th>
+                        <th>Artists</th>
+                        <th colspan="2" class="text-center">Actions</th>
+                    </tr>
+                    </thead>
+                    @foreach (var song in Model.Songs)
+                    {
+                        <tr>
+                            <td>@song.Title</td>
+                            <td class="duration-cell">@($"{song.Duration / 60:D2}:{song.Duration % 60:D2}")</td>
+                            <td>@song.ReleaseDate.ToShortDateString()</td>
+                            <td>@song.Genre?.Name</td>
+                            <td>@string.Join(", ", song?.Artists ?? Enumerable.Empty<Domain.Models.Artist>())</td>
+                            <td class="action-buttons">
+                                <a asp-page="./UpdateSong" asp-route-id="@song?.Id">Update</a>
```

Comment:
How song could be null here? Only if you have the collection of Song? (I mean List<Song?>)